modules/serverless-customize-labels-annotations-routes.adoc

```diff
@@ -0,0 +1,52 @@
+[id="serverless-customize-labels-annotations-routes_{context}"]
+= Customizing labels and annotations for {product-title} routes
+
+{product-title} routes support the use of custom labels and annotations, which you can configure by modifying the `metadata` spec of a Knative service. Custom labels and annotations are propagated from the service to the Knative route, then to the Knative ingress, and finally to the {product-title} route.
+
+.Prerequisites
+
+* You must have the {ServerlessOperatorName} and Knative Serving installed on your {product-title} cluster.
+
+.Procedure
+
+. Create a Knative service that contains the label or annotation that you want to propagate to the {product-title} route.
+** Create a service by using YAML:
++
+.Example service created by using YAML
+[source,yaml]
+----
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: <service_name>
+  labels:
+    <label_name>: <label_value>
+  annotations:
+    <annotation_name>: <annotation_value>
+...
+----
+** Create a service by using the `kn` CLI:
++
+.Example service created by using a `kn` command
+[source,terminal]
+----
+$ kn service create <service_name> \
+  --image=<image> \
+  --annotation <annotation_name>=<annotation_value> \
+  --label <label_value>=<label_value>
+----
+
+. Verify that the {product-title} route has been created with the annotation or label that you added by inspecting the output from the following command:
```

Comment:
Is there anything in particular that I should look for in the output? I assume I'm looking for a route that has the annotation or label. Could note that right after the example command + callouts.

Reply:
Just that the label or annotation is there, I think this is OK as is since it'll depend what labels or annotations a user was adding.